Config.ts

```diff
@@ -1,4 +1,8 @@
-export default {
-  network: process.env.REACT_APP_NETWORK ?? 'fuse',
-  linkUrl:"http://wallet.gooddollar.org:3000/?code="
-}
\ No newline at end of file
+import ContractsAddress from '@gooddollar/goodprotocol/releases/deployment.json'
+const config =  {
+  networkId: ContractsAddress['fuse'].networkId,
+  linkUrl:"http://wallet.gooddollar.org:3000/?code=",
+  network:"production-mainnet",
+  web3RPC:"https://rpc.fuse.io",
```

Comment:
if we are doing it hard coded then
networkId should be 122 and network should be production

Reply:
Okay